src/common/sensor.rs

```diff
@@ -0,0 +1,48 @@
+#![allow(dead_code)]
+
+use crate::common::status::Status;
+use std::collections::BTreeMap;
+
+type SensorReadings = ::std::collections::HashMap<String, ::prost_types::Value>;
```

Comment:
@npmenard

- I'm not sure if the type alias is worthwhile, but I was getting tired of typing `std::collections::HashMap`.
- I don't particularly like having `prost_types::Value` leaking into the user interface here. I suspect that this should be `f64`, but I was having trouble elsewhere when I did that.

Reply:
Type aliases a nice to make the code more readable but I agree we need to hide Value from the user since anyone should be able to use this interface.
`f64` make sense here. Also you should probably let each sensor implement their own version of Readings ( in go a Reading is a `map[string]interface{}`